digital_appointment/medicine/models.py

```diff
@@ -0,0 +1,29 @@
+from django.db import models
+from accounts.models import User
+
+
+class Service(models.Model):
+    name = models.CharField(max_length=100)
+    description = models.TextField()
+    duration = models.DurationField()
```

Comment:
The purpose of creating the duration field is to determine a specified time for the end of a task. If I used the integer field, I would have to set a series of configurations for this field, but this field uses the timedelta method and it is the best in terms of optimization. The input of this field is in the format of time, and if it is entered as a single number, it will be converted into seconds!